neo/VM/JsonParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Neo.IO.Json;
+using System;
+using System.Numerics;
+using System.Text;
+
+namespace Neo.VM
+{
+    public static class JsonParser
+    {
+        /// <summary>
+        /// Convert json object to stack item
+        /// </summary>
+        /// <param name="json">Json</param>
+        /// <returns>Return stack item</returns>
+        public static StackItem JsonToStackItem(JObject json)
+        {
+            switch (json)
+            {
+                case JArray array:
+                    {
+                        var item = new Types.Array();
+
+                        foreach (var entry in array)
+                        {
+                            item.Add(JsonToStackItem(entry));
+                        }
+
+                        return item;
+                    }
+                case JString str:
+                    {
+                        return new Types.ByteArray(Encoding.UTF8.GetBytes(str.Value));
+                    }
+                case JNumber num:
+                    {
+                        return new Types.Integer((BigInteger)num.Value);
```

Comment:
num.Value is safe against bigint? Is it long? I'll check. Looks good.

Reply:
if is very high, maybe you can get wrong results, maybe is better to check it for return string or integer...

Reply:
The `number`s in JavaScript are IEEE-754 floating point numbers.

Reply:
So do you think that is better to convert it to ByteArray?

Reply:
Throws if it's not an integer.